### PR TITLE
show GEN until boosted

### PR DESCRIPTION
**Pull request by dev-matan-tsuberi**
**Note: Needs some styling.**

### CODE REVIEW

**Comment by dev-matan-tsuberi** [comment]
[Indicate on a proposal how much staking would get the proposal boosted](https://trello.com/c/eqS8bRD5/122-indicate-on-a-proposal-how-much-staking-would-get-the-proposal-boosted)

File: src/actions/arcActions.ts [hunk 1]
```diff
@@ -314,6 +314,7 @@ async function getProposalDetails(dao: Arc.DAO, votingMachineInstance: Arc.Genes
     votesYes: Util.fromWei(yesVotes).toNumber(),
     votesNo: Util.fromWei(noVotes).toNumber(),
     winningVote: Number(proposalDetails[9]),
+    threshold: Util.fromWei(new BigNumber(await votingMachineInstance.getThreshold({avatar: dao.avatar.address, proposalId}))).toNumber()
```
[the remark this answers is not in the document]
Comment (dev-matan-tsuberi) [reply src/actions/arcActions.ts]: `number`

[the remark this answers is not in the document]
Comment (tibetsprague) [reply src/actions/arcActions.ts]: how does it return WEI as a number?

[the remark this answers is not in the document]
Comment (dev-matan-tsuberi) [reply src/actions/arcActions.ts]: The type says `number`.
@dkent600 Could be an Arc.js issue?

[the remark this answers is not in the document]
Comment (dev-matan-tsuberi) [reply src/actions/arcActions.ts]: here's the issue: https://github.com/daostack/arc.js/issues/213
merging this PR and we'll change it once resolved and published.

**Pull request merged**